tests/unit/amo/sagas/testLanding.js

```diff
@@ -116,4 +122,116 @@ describe('amo/sagas/landing', () => {
       expect(calledActions[1]).toEqual(errorAction);
     });
   });
+
+  describe('getLandingForCategory', () => {
+    const addonType = ADDON_TYPE_EXTENSION;
+    const category = 'some-category';
+
+    let errorHandler;
+    let mockApi;
+    let mockSearchApi;
+    let sagaTester;
+
+    beforeEach(() => {
+      errorHandler = createStubErrorHandler();
+      mockApi = sinon.mock(api);
+      mockSearchApi = sinon.mock(searchApi);
+      sagaTester = new SagaTester({
+        initialState: dispatchClientMetadata().state,
+        reducers: { landing: landingReducer, api: apiReducer },
```

Comment:
You did https://github.com/mozilla/addons-frontend/pull/3343/files#diff-c339389f32102f7b6fc32a1dd8062aabR29 which I liked but then here it's on one line and not even sorted. 😅